### DISCUSSION TITLE
Should I use 1 Nuxt app for my website and admin or 2 Nuxt?

### DISCUSSION THREAD

**Discussion by lucassimines** [nuxt/nuxt#17734] [Questions]
I'm developing a complete `/admin` for managing a `website frontend`, what do you recommend, having 2 Nuxt instalations separated or only 1, for both `/admin` and `website frontend`? Thanks!

**Comment by vis97c** [comment]
You could create a single nuxt app  and use a different layout for each section. You would also be able to reuse a lot of the logic.

**Comment by mihail727** [comment]
there is a legend that it is safer this way - when you separate the admin panel and the site itself into different projects.
but it usually depends on the task

for example, if your admin panel can manage several sites, then it definitely needs to be moved to a separate project

As for me, the safest solution, but the fattest one is to divide the project into 3 sites:
- first is default site - for client
- second admin panel - for data management
- the third so-called data center - only he will have direct access to the database, only he will act as an rest api

in the end it all depends on the scope of the project and the goals of the project

**Reply by lucassimines** [reply]
> there is a legend that it is safer this way - when you separate the admin panel and the site itself into different projects. but it usually depends on the task
> 
> for example, if your admin panel can manage several sites, then it definitely needs to be moved to a separate project
> 
> As for me, the safest solution, but the fattest one is to divide the project into 3 sites:
> 
> * first is default site - for client
> * second admin panel - for data management
> * the third so-called data center - only he will have direct access to the database, only he will act as an rest api
> 
> in the end it all depends on the scope of the project and the goals of the project

My admin just manage 1 site.
And I was thinking into diving it in 3 sites as you mentioned.

Thanks @vis97c and @mihail727

**Reply by mihail727** [reply]
ur welcome
